### PR TITLE
Cleaned up map_data

### PR DESCRIPTION
**Pull request by lxgr-linux**

### CODE REVIEW

File: pokete_data/maps.py [hunk 1]
```diff
@@ -379,6 +379,13 @@
         "extra_actions": None,
         "poke_args": None
     },
+    "playmap_39": {
+        "height": 69,
+        "width": 144,
+        "pretty_name": "Agrawos",
+        "extra_actions": None,
+        "poke_args": None
+    },
```
Comment (MaFeLP) [inline pokete_data/maps.py:388]: Doesn't there have to be a definition of the map in `pokete_data/map_data.py`?

Comment (lxgr-linux) [reply pokete_data/maps.py:388]: Yes, I planned to do that. This is a map that will be added to the game later.

File: pokete_data/map_data.py [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+|___| |___|__|"""
+
+house1 = r"""  ________
+ /       /\
+/_______/  \
+|# ___ #|  |
+|__| |__|__|"""
+
+
+# This is the the actuall map_data dict
```
Comment (MaFeLP) [inline pokete_data/map_data.py:25]: Trailing whitespace, doubled word and typo:
```suggestion
# This is the actual map_data dict
```

File: pokete_data/map_data.py [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+/__Shop!__/  \
+| # ___ # |  |
+|___| |___|__|"""
+
+house1 = r"""  ________
+ /       /\
+/_______/  \
+|# ___ #|  |
+|__| |__|__|"""
+
```
Comment (MaFeLP) [inline pokete_data/map_data.py:23]: These are constants, so it would be better write them in upper case

**Review by MaFeLP — CHANGES_REQUESTED** [review]

**Review by MaFeLP — APPROVED** [review]

**Pull request merged**